packages/notification/docs/InlineNotification.stories.tsx

```diff
@@ -97,6 +97,18 @@ const StickerSheetTemplate: Story<{ isReversed: boolean }> = ({
             <a href="/">View all</a>
           </InlineNotification>
         </StoryWrapper.Row>
+        <StoryWrapper.Row rowTitle="Subtle">
+          <InlineNotification
+            type="informative"
+            isSubtle
+            title="Informative title"
+            hideCloseIcon
+          >
+            "All Employees - North America" status has been changed to
+            'Archived'.
+            <a href="/">View all</a>
```

Comment:
It's `children`, but the story looks like a space is missing so probably better to just add it 🤷🏻 
```suggestion
            "All Employees - North America" status has been changed to
            'Archived'. <a href="/">View all</a>
```